src/index.js

```diff
@@ -34,6 +34,9 @@ module.exports = class OpenApiValidator {
                     filePath: this.apiSpec,
                 });
                 this.apiDoc = await loader.getDoc();
+
+                // Event body is in a string format by default, need to convert to JSON
+                event.body = JSON.parse(event.body);
```

Comment:
What happens if the event.body isn't JSON? Are we cool just throwing an error about that?

Reply:
Once API Lambda is being used it would always be a string. Throwing the default error might just enforce that.

Reply:
Updated for this use case, so it'll accept any type in the event body